presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotPageSourceProvider.java

```diff
@@ -127,4 +131,20 @@ public ConnectorPageSource createPageSource(
                 throw new UnsupportedOperationException("Unknown Pinot split type: " + pinotSplit.getSplitType());
         }
     }
+
+    @VisibleForTesting
+    static GrpcQueryClient.Config extractGrpcQueryClientConfig(PinotConfig config) {
+        Map<String, Object> target = new HashMap<>();
+        target.put(CONFIG_USE_PLAIN_TEXT, !config.isUseTlsForGrpc());
+        target.put(CONFIG_MAX_INBOUND_MESSAGE_BYTES_SIZE, config.getStreamingServerGrpcMaxInboundMessageBytes());
+        if (config.isUseTlsForGrpc()) {
+            target.put("tls.keystore.path", config.getGrpcTlsKeyStorePath());
```

Comment:
Seems this one is needed in order to talk to Pinot Proxy. Does not matter what it is inside because we don't do two-way TLS.

Reply:
(Maybe I should try some test without any of these configs, and see if it goes through)